app/src/main/java/com/mashup/lastgarden/ui/onboarding/OnBoardingActivity.kt

```diff
@@ -2,13 +2,51 @@ package com.mashup.lastgarden.ui.onboarding
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import com.mashup.lastgarden.R
+import androidx.viewpager2.widget.ViewPager2
+import com.google.android.material.tabs.TabLayoutMediator
+import com.mashup.lastgarden.databinding.ActivityOnBoardingBinding
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class OnBoardingActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityOnBoardingBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_on_boarding)
+        binding = ActivityOnBoardingBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        setupViewPager()
+
+        binding.vpOnboarding.registerOnPageChangeCallback(object :
+            ViewPager2.OnPageChangeCallback() {
+
+            override fun onPageSelected(position: Int) {
+                super.onPageSelected(position)
+                when (binding.vpOnboarding.currentItem) {
+                    3 -> binding.btnNext.text = "시작하기"
+                    else -> binding.btnNext.text = "다음"
+
+                }
+            }
+        })
+    }
+
+    private fun setupViewPager() {
+        binding.vpOnboarding.adapter = TutorialViewPagerAdapter(this)
+        TabLayoutMediator(binding.tabLayout, binding.vpOnboarding) { tab, _ ->
+        }.attach()
```

Comment:
tab 안쓰니까 _ 로 받아도 될 것 같아요 
```suggestion
        TabLayoutMediator(binding.tabLayout, binding.vpOnboarding) { _ , _ ->
        }.attach()
```